state-network.md

```diff
@@ -125,6 +125,15 @@ A node should track their own radius value and provide this value in all Ping or
 
 ### Data Types
 
+#### Component Data Elements
+
+#### Proofs
+Merkle Patricia Trie (MPT) proofs consist of a list of witness nodes that correspond to each trie node that consists of various data elements depending on the type of node (e.g.blank, branch, extension, leaf).  When serialized, each witness node is represented as an RLP serialized list of the component elements with the largest possible node type being the branch node which when serialized is a list of up to sixteen hashes in `Bytes32` (representing the hashes of each of the 16 nodes in that branch and level of the tree) plus the 4 elements of the node's value (balance, nonce, codehash, storageroot) represented as `Bytes32`.  When combined with the RLP prefixes, this yields a possible maximum length of 667 bytes.
+```
+mpt_witness_node       := ByteList(667)
```

Comment:
I may be wrong here, but I think that `ByteList(max_length=X)` is just shorthand for `List[uint8, max_length=XXX]` and SSZ only allows for lists with lengths that are powers of 2, which suggests that we need to bump this to `ByteList(1024)`

Reply:
Good point.  Forgot about that when I was drafting this and our ssz implementation didn't complain when I coded this locally so will update here.

Reply:
SSZ does allow for any max_length. It is just that when doing merkleization chunks will be padded to the next_pow_of_two(max_length). But you can still limit the actual list to 667 here.